synapse/config/tracer.py

```diff
@@ -49,16 +49,6 @@ def generate_config_section(cls, **kwargs):
 
             # The list of homeservers we wish to send and receive span contexts and span baggage.
```

Comment:
Can you add a reference to the rst file in the introductory comment? (I think we may do this for other things in the config file - see if you can follow the style)